### ISSUE TITLE
Steam and Xbox game pass inconsistent auto-importing

### ISSUE THREAD

**Issue by jamie9090** [JosefNemec/Playnite#1963]
Xbox PC Game Pass and Steam occasionally have issues with automatic imports. No message added, it just doesn't automatically import games. 

Originally, they were working fine.

I have found the issue with Steam only started after I changed my install location to another disk drive, then re-downloaded CS:GO and Rainbow Six Siege. Those are my only STEAM games. To re-add them, I had to manually add via Playnite.

The issue with Xbox Game Pass started with Middle-earth Shadow of Modor. It then also occured with Master Chief Collection after reinstall, as originally it was working.

3e53b063-549e-4a50-b2a9-bab4912b3a2b

**Comment by darklinkpower** [comment]
Check this and see if it helps. For Steam you'll probably need to reinstall it to fix the issues.
https://github.com/JosefNemec/Playnite/wiki/Steam-troubleshooting
https://github.com/JosefNemec/Playnite/wiki/Xbox-Library-troubleshooting

For Xbox, verify if the games have been started at least once and that you are authenticated in the plugin settings. There's also this fix that will be released in a future update that will probably fix your issue if it still doesn't work: https://github.com/JosefNemec/Playnite/commit/3928ac0293a73de461a118d68b34a27d8bdbb09b

In the meantime you can add the import the games manually from `Main Menu -> Add Game`

**Comment by jamie9090** [comment]
Thank you!

**Comment by jamie9090** [comment]
RE: Steam 

It happended after I changed Steam Library location via Steam's settings (eg: Change from c:\ to g:\). Could the issue be linked to that?

If I reinstalled STEAM and changed the drive, the issue may then repeat? Not sure if others have been able to do that fine.

**Comment by JosefNemec** [comment]
Xbox issue has been fixed in 7.9 update.

If you are having issue with Steam import, then reinstall Steam client. It fixed the issue in 99% of cases.

**Comment by spacepanda** [comment]
Hello, I am having a similar issue to Jamie and I can confirm a reinstall did not fix. Issue began when I installed Steam games to my E drive in a different location from my main steam library, none of the games installed on E are detected as installed by Playnite despite working fine on steam.

**Comment by JosefNemec** [comment]
Did you reinstall Steam or just Steam games? You need to reinstall Steam client, that's the only way to fix it.

**Comment by spacepanda** [comment]
Client, not games.

**Comment by wtfatty** [comment]
> Xbox issue has been fixed in 7.9 update.

Hello, about the Xbox Game Pass games, I'm viewing only the ones I've launched at least once. 
I don't know if this is intended, but it would be great to have all the available games.

**Comment by JosefNemec** [comment]
Game Pass games will be imported only after launched at least once.

**Comment by wtfatty** [comment]
> Game Pass games will be imported only after launched at least once.

Yeah, too bad :(
GoG has an option to show both Owned Games and Subscriptions game.

**Comment by darklinkpower** [comment]
I'm having this issue with Disgaea 4 Complete from Gamepass. Game has been launched once, account is authenticated.
Xbox app launches correctly from Playnite. I tested with another game "Downwell" and it imported fine so it seems the issue is not global.

Diagnostic package: b54b54bc-7049-4ba3-a89d-d1ea32191c54

**Comment by darklinkpower** [comment]
As an update, for no apparent reason the game that was not being imported got correctly added after a few days.

**Comment by JosefNemec** [comment]
Closing, if you have some other issues with specific games, open new issue for it.
